game.project

```diff
@@ -28,31 +28,35 @@ admob_android_appid = ca-app-pub-3940256099942544~3347511713
 applovin_android = 1
 bidmachine_android = 1
 charboost_android = 1
-fyber_android = 1
+dt_exchange_android = 1
 facebook_android = 1
 hyprmx_android = 1
 inmobi_android = 1
 liftoff_android = 1
 mintegral_android = 1
+moloco_android = 1
 mytarget_android = 1
 pangle_android = 1
 unityads_android = 1
+yandex_ads_android = 1
 ios_use_skan = 1
 ios_tracking_usage_description = Your data will be used to provide you a better and personalized ad experience.
 ios_enable_ats = 1
 adcolony_ios = 1
 admob_ios = 1
 admob_ios_appid = ca-app-pub-3940256099942544~1458002511
 applovin_ios = 1
+bidmachine_ios = 1
 charboost_ios = 1
-fyber_ios = 1
+dt_exchange_ios = 1
 facebook_ios = 1
 hyprmx_ios = 1
 inmobi_ios = 1
 liftoff_ios = 1
 maio_ios = 1
 mintegral_ios = 1
-mytarget_ios = 1
```

Comment:
unf. this network can't be used because of https://github.com/defold/extender/issues/388 
Can be re-added when we fix it